spec/controllers/media_objects_controller_spec.rb

```diff
@@ -259,8 +259,6 @@
           # master_file_obj = FactoryBot.create(:master_file, master_file.slice(:files))
           media_object = FactoryBot.create(:media_object)#, master_files: [master_file_obj])
           fields = media_object.export_descriptive_metadata_attributes
-          descMetadata_fields.each {|f| fields[f] = media_object.send(f) }
-          # fields = media_object.attributes.select {|k,v| descMetadata_fields.include? k.to_sym }
```

Comment:
Removed as `media_object.export_descriptive_metadata_attributes`, a locally created function, completes the same function plus more (related to language and PBcore as a datasource). The highlighted is repeated a total of four times in the file

Links to the PR #302 and commit
https://github.com/ualbertalib/avalon/commit/3585bb0212f5032838d4f6d79a13fded9818e900
3585bb0